e2etest/bgptests/bgp.go

```diff
@@ -66,6 +66,7 @@ var (
 	Reporter            *k8sreporter.KubernetesReporter
 	ReportPath          string
 	PrometheusNamespace string
+	NativeMode          bool
```

Comment:
I'd rather drive the skip from outside, or prepend the table entry with FRR so it will be skipped.

Reply:
done